pkg/datastore/eventstore.go

```diff
@@ -49,6 +49,7 @@ func NewEventStore(ds DataStore) EventStore {
 }
 
 func (s *eventStore) AddEvent(ctx context.Context, e model.Event) error {
+	e.EventKey = model.MakeEventKey(e.Name, e.Labels)
```

Comment:
I think it is better to set the key when making the event instead of here.
https://github.com/pipe-cd/pipe/blob/master/pkg/app/api/grpcapi/api.go#L323

Reply:
Every event should have an event-key derived from the exact same method. So I thought making here is more appropriate, but you don't think so right?

Reply:
Yes, I got your point. I was thinking the same.
But I feel updating the model inside this function is not good because the store is just for storing and querying.

(To be honest, I also want to remove the part of updating CreatedAt and UpdatedAt from this function. But that is future talk.)

Reply:
Fine, that makes total sense. This `datastore` package provides non-domain-specific, versatile facilities and should just operate external Datastore services. I feel exactly the same, so let me fix it!